src/main.rs

```diff
@@ -171,6 +172,17 @@ async fn schedule_background_jobs(
     .unwrap();
     jobs.push(event_pruning_job);
 
+    let snapshot_upload_job = snapchain::jobs::snapshot_upload::snapshot_upload_job(
+        "0 0 5 * * *", // 5 AM UTC every day
```

Comment:
We should only do this if snapshots are enabled.

Reply:
Good point, fixed.